### PR TITLE
Update easy.yaml with many informative comments and include meta-ignores for difficult modes

### PR DESCRIPTION
**Pull request by LegendaryLinux**

### CODE REVIEW

File: easy.yaml [hunk 1]
```diff
@@ -1,160 +1,171 @@
...
+name: YourName # Your name in-game. Spaces and underscores will be replaced with dashes
+glitches_required: # Determine the logic required to complete the seed
+  none: 1 # No glitches required
+  overworld_glitches: 0 # Assumes the player knows how to perform overworld glitches like fake flipper, water walk, etc
+  no_logic: 0 # Items are places completely at random and with no regard for logic. Your fire rod could be on Trinexx
+item_placement: basic # This is based on Entrance Randomizer, which does not (yet?) support advanced
+meta_ignore: # Nullify options specified in the meta.yaml file. Adding an option here guarantees it will not occur in your seed, even if the .yaml file specifies it
+  world_state:
+    - inverted # Never play inverted seeds
+    - retro # Never play retry seeds
```
Comment (Berserker66) [inline easy.yaml:30]: typo

File: easy.yaml [hunk 2]
```diff
@@ -1,160 +1,171 @@
...
+#   off: 15
+#
+# Means you have 5 chances for map shuffle to not occur, and 15 chances for map shuffle to be turned on
+
+# I've never seen a file like this before. What characters am I allowed to use?
+# This is a .yaml file. You are allowed to use most characters.
+# To test if your yaml is valid or not, you can use this website:
+# http://www.yamllint.com/
+
+description: Farrak's Easy Example # Used to describe your yaml. Useful if you have multiple files
```
Comment (Berserker66) [inline easy.yaml:20]: I'd prefer to keep it as enter name here or something to that extent, as this is the most easily forgotten line

**Comment by LegendaryLinux** [comment]
I updated the default description name and fixed the typo.

File: easy.yaml (hunk 1 above)
Comment (LegendaryLinux) [reply easy.yaml]: Typo fixed in new commit.

File: easy.yaml (hunk 2 above)
Comment (LegendaryLinux) [reply easy.yaml]: Description updated in new commit.

File: easy.yaml [hunk 3]
```diff
@@ -1,160 +1,171 @@
...
+  items: 0 # Guarantees you will be able to acquire all items, but you may not be able to access all locations
+  locations: 1 # Guarantees you will be able to access all locations, and therefore all items
+  none: 0 # Guarantees only that the game is beatable. You may not be able to access all locations or acquire all items
+progressive: # Enable or disable progressive items (swords, shields, bow)
+  on: 1 # All items progressive
+  off: 0 # No items progressive
+  random: 0 # Randomly decides for all items. Swords could be progressive, shields might not be
+entrance_shuffle: # Documentation: https://alttpr.com/en/options#entrance_shuffle
+  none: 1 # Vanilla game map. All entrances and exits lead to their original locations. You probably want this option
+  simple: 0 # Entrances are grouped together before being randomized. Simple uses the most strict grouping rules
```
Comment (CaitSith2) [inline easy.yaml:55]: dungeonssimple and dungeonsfull are missing from the options.

**Pull request merged**